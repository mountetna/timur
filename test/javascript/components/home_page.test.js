import React from 'react';
import {Provider} from 'react-redux';
import {mount, shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import {mockStore} from '../helpers';
import HomePage from '../../../lib/client/jsx/components/home_page';

const permissions = require('../fixtures/home_page_permissions.json');
const projects = require('../fixtures/project_names.json')

describe('HomePage', () => {
  it('renders', () => {
    const tree = renderer
      .create(<HomePage permissions={permissions} projects={projects} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
