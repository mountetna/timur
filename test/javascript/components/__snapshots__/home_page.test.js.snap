// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders 1`] = `
<div
  className="browser project"
>
  <div
    className="home-page-header-group"
  >
    <div
      className="page-detail-group"
    >
      Available Projects
    </div>
  </div>
  <div
    className="browser-tab"
  />
</div>
`;
